### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Sammy3000**
## In this Pull request I did the following. Build my Worldwide app while following the instructions below:
- Chose an open API(Rest Countries) to build my app around.
### Home page

- When the page loads, the webapp shows the list of categories that could be filtered out by some parameter.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.
### Details page

In the details page, the webapp retrieves data from the API to show detailed data in the category.
When the user clicks on the "Back" button (<), the user navigates to the home page.

### Testing requirements

Create unit tests for pure functions in your Redux code.
Create integration tests for your application using the React Testing Library.
You may need to mock the access to the API, so that your tests don't send actual requests.
You may need to mock the connection to the Redux Store.

### Technical requirements

The project is a single-page application (SPA) built with React and Redux.
The data retrieved from the API should be stored in the Redux store.
You should filter the data that you retrieve from the API using a Filter stateless component.
Every page (the main page and the pages for each item) should have a unique route within the SPA.
The project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/).
Add a link to the online version of your app to your README file.

### Project documentation
Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

a description of the project.
a demo of the project features.
you should also highlight some interesting piece of code or something you built that you are very proud of.
For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,6 @@
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
-- [📖 Space Mission App ](#-space-mission-app-)
   - [🛠 Built With ](#-built-with-)
```
Comment (ghost) [inline README.md:8]: image

- [ ] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] The link to your `MIT` License raises a `404` page error when visited, kindly adjust the pathname to make your readme more professional

File: src/features/countries/countriesSlice.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// import axios from 'axios';
```
Comment (ghost) [inline src/features/countries/countriesSlice.js]: - [ ] Kindly remove commented our code to keep your code base clean

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Sammy3000 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)


## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅All tests are passing
✅ App is designed as expected 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sammy3000** [comment]
- Thank you @frank1738 . I will do the changes.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Sammy3000 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done on making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sammy3000** [comment]
- Thank you @MahmoudBakr23 .

**Comment by Sammy3000** [comment]
Completed successfully.

**Pull request closed without merging**